packages/thread-comment/src/controllers/tc-remove-subunit.controller.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Workbook } from '@univerjs/core';
+import { Disposable, IUniverInstanceService, LifecycleStages, OnLifecycle, UniverInstanceType } from '@univerjs/core';
+import { Inject } from '@wendellhu/redi';
+import type { IRemoveSheetCommandParams } from '@univerjs/sheets';
+import { RemoveSheetCommand, SheetInterceptorService } from '@univerjs/sheets';
```

Comment:
If this package is for general purposes, it shoud not rely on `@univerjs/sheets`.

Reply:
moved to sheets-thread-comment